pytest_fixtures/component/taxonomy.py

```diff
@@ -141,6 +141,15 @@ def upgrade_entitlement_manifest_org(function_org, upgrade_entitlement_manifest,
     return function_org
 
 
+@pytest.fixture
+def sca_manifest_org_for_upgrade(function_org, sca_manifest_for_upgrade, target_sat):
```

Comment:
would it be ok to change the name something like `fun_sca_manifest_org_for_upgrade` or `function_manifest_org_for_upgrade` for better understanding of fixture scope

Reply:
I already changed the name for @jyejare. Lets keep it as is